development/tools/pkg/pjtester/pjtester.go

```diff
@@ -205,18 +210,23 @@ func (o options) withGithubClientOptions() options {
 
 // getPullRequests will download pull requests details from github, for numbers provided in pjtester.yaml.
 func (o *options) getPullRequests(t testCfg) {
-	o.pullRequests = t.PrConfigs
+	if o.pullRequests == nil {
+		o.pullRequests = make(map[string]prOrg)
+	}
 	for org, repos := range t.PrConfigs {
+		if _, ok := o.pullRequests[org]; !ok {
+			o.pullRequests[org] = prOrg{}
+		}
 		for repo, prcfg := range repos {
 			pr, err := o.githubClient.GetPullRequest(org, repo, prcfg.PrNumber)
 			if err != nil {
-				logrus.WithError(err).Fatalf("failed to fetch PullRequest from GitHub: %v", err)
+				logrus.WithError(err).Fatalf("failed to fetch PullRequest from GitHub")
```

Comment:
This is the package-level function. I would rather rewrite the package functions, even if they are unexported, to return the `error` instead of failing the entire program. If we were to use this package in different project then this might be problematic. That applies to all `Fatals` in the package.

Reply:
When we will need use it in another project then we can rewrite this. Now it's a standalone tool.

Reply:
Yes, I understand that but this is the pattern that we should likely avoid, if we are talking about packages. Additionally it will be much easier to write tests for these methods if they return an `error` value.

Reply:
Tests are already written.